hack/install.sh

```diff
@@ -47,6 +47,14 @@ function download_ui() {
   rm -rf "${tmp_ui_path}"
 }
 
+# Copy extra static files to ui including catalog icons
+function copy_extra_static() {
+  local extra_static_path="${ROOT_DIR}/static"
+  local ui_static_path="${ROOT_DIR}/gpustack/ui/static"
+  if [ -d "${extra_static_path}" ]; then
+    cp -a "${extra_static_path}/." "${ui_static_path}"
```

Comment:
Will git changes be introduced here, dirty changes will cause invalid released version for docker images

Reply:
UI is isgnored: https://github.com/gpustack/gpustack/blob/main/.gitignore#L154